packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx

```diff
@@ -35,10 +35,20 @@ export const IdentifierInput: React.FC<IdentifierInputProps> = ({ patientIdentif
   });
   const autoGenerated = !option.manualEntryEnabled || (option.manualEntryEnabled && option.automaticGenerationEnabled);
   const isDisabled = isOffline && shouldBlockPatientIdentifierInOfflineMode(identifierType);
```

Comment:
For `isDisabled` the 'is' is redundant. You could call the variable `disabled` and it would mean the same thing, or you could be more specific and use `editButtonDisabled` which can better help understand what it's supposed to do.

https://hilton.org.uk/blog/naming-smells#vague-words

Reply:
Not sure I agree with you here, @ZacButko . I have seen tons of OpenMRS code use variables called `active` or `disabled` to hold non-boolean data (e.g., the active or disabled component itself). It is terrible naming but extremely common in OpenMRS code. In this context, the `is-` prefix helpfully signals that the variable is most definitely a boolean. See also https://github.com/kettanaito/naming-cheatsheet#prefixes . I do agree with you that simply `active` or `disabled` are preferable in APIs where only a single entity is concerned, e.g. `<Link to="home" active />`.